rskj-core/src/main/java/co/rsk/peg/bitcoin/BitcoinUtils.java

```diff
@@ -57,7 +57,17 @@ public static Optional<Script> extractRedeemScriptFromInput(TransactionInput txI
         }
     }
 
-    public static void removeSignaturesFromTransactionWithP2shMultiSigInputs(BtcTransaction transaction) {
+    public static Sha256Hash getMultiSigTransactionHashWithoutSignatures(NetworkParameters networkParameters, BtcTransaction transaction) {
+        if (!transaction.hasWitness()) {
```

Comment:
```suggestion
        if (transaction.hasWitness()) {
           return transaction.getHash();
        }
        
        ...
```
Nit, just to return early

Reply:
i put it this way considering the txs are normally segwit (correct me if im wrong! :) ), so we will have the common behavior outside the check